packages/sodium/src/js/modules/SessionData.js

```diff
@@ -503,26 +504,25 @@ export default class SessionData {
       };
       window.addEventListener("message", listener);
       window.postMessage({
-        host: url.host,
+        host: this.location.host,
         method: "get_ip",
         type: "FROM_SODIUM_JS"
       });
     });
-    this.hostToIp[url.host] = ip;
+    this.hostToIp[this.location.host] = ip;
   }
 
   altSessionMessage() {
     const allowHosts = ["tver.jp", "fod.fujitv.co.jp"];
     const allowVideoHosts = ["i.fod.fujitv.co.jp"];
 
     if (window.top === window) {
-      if (!allowHosts.includes(window.location.hostname)) return;
+      if (!allowHosts.includes(this.location.host)) return;
```

Comment:
```suggestion
      if (!allowHosts.includes(this.location.hostname)) return;
```
URL.hostとURL.hostnameでは意味が異なるように思うのですが問題ないですか?

Reply:
https://developer.mozilla.org/ja/docs/Web/API/URL/host
https://developer.mozilla.org/ja/docs/Web/API/URL/hostname

Reply:
失礼しました。その通りですね。戻しました。